azkaban-common/src/test/resources/flow_filter.txt

```diff
@@ -0,0 +1,3 @@
+proj1:flow1
```

Comment:
I am curious how often these configurations get updated. This is a well-structured configuration and we have provided a way to reload the configuration as necessary in this PR. Did we consider keeping this config in our database?

Reply:
We expect to update the config by directly modifying the file either using some script or manually but not frequently. Once modified, we call the reload API which updates the filter in-memory.